portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,28 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#sidebar {
+    height: 100%;
+    width: 200px;
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    padding-top: 20px;
+    background-color:black;
+}
```

Comment:
CSS best practice is to alphabetically order properties.

Reply:
Got it! Reordering these right now.